npm-install-subfolders.js

```diff
@@ -0,0 +1,48 @@
+const { execSync } = require('child_process');
+const { readdirSync, statSync } = require('fs');
+const { join, resolve } = require('path');
+
+
+const initialDir = process.cwd(); // Store the initial working directory
+
+function findDirectSubfolders(dir) {
```

Comment:
There may be widgets in the subfolders, so I think we need to search the subfolders recursively to find all of them. When we search subfolders recursively, we can check the `package.json` and `manifest.json`, but we need to make sure skip the `node_modules` folder.

Reply:
Hi @qlqllu, oh I see. I accidentally commited two changes here. This should have only been the [README](https://github.com/Esri/arcgis-experience-builder-sdk-resources/pull/184/commits/c48eeaf68c1a7428b91f1273ad52b7c85f16c8f2) updates.
The one you're commenting on actually refers to [PR Npm install subfolders #174](https://github.com/Esri/arcgis-experience-builder-sdk-resources/pull/174).